### ISSUE TITLE
Acceptance Test AT3-7

### ISSUE THREAD

**Issue by MaelSemler** [MaelSemler/mini-cap#172]
**Acceptance test:** 

AT-3-7

**Feature:** 

FT-3-7

**Feature description:** 

 As a user I want the directions to my next class from my current location. 

**Acceptance criteria:**

Steps to perform acceptance test, with each on a different line, e.g.

Given the app is open,
From the hamburger menu,
I can click on “Schedule”,
And I can connect to my Google account, 
I can then click on Next Class,
Then, I should be able to see directions to my next class.

**Result:**

To be filled in by product owner upon evaluation.

**Date:**

To be filled in by product owner upon evaluation.

**Signature:**

To be filled in by product owner upon evaluation.

**Comment by mkshiblu** [comment]
Great job on the UI part. Looks very professional.

What is the proper format of the location?


![image](https://user-images.githubusercontent.com/15908043/79893915-b7e79600-83d2-11ea-90fb-d53f5ab99adb.png)

**Comment by MaelSemler** [comment]
Hi @mkshiblu , a proper location would be in that format, for outdoor directions: John Molson School of Business, 1450 Rue Guy, Montréal, QC H3H 0A1, Canada

**Comment by mkshiblu** [comment]
Thanks, after hitting go now it focuses on the user location. Is this expected?

![image](https://user-images.githubusercontent.com/15908043/80028258-d755f000-84b2-11ea-9af2-b75c4a8d9b57.png)

**Comment by MaelSemler** [comment]
@mkshiblu Yes, and then you can click on the arrow at the bottom to show the directions

**Comment by mkshiblu** [comment]
ACCEPTED. Thank you.
